src/content/docs/security/security-privacy/compliance/fedramp-compliant-endpoints.mdx

```diff
@@ -525,4 +525,9 @@ To use the Log API directly, you'd edit the [Log API endpoint configuration](/do
 
 ### Trace API [#trace-api]
 
-FedRAMP is not yet available for the Trace API itself, but note that some of our agents report trace data and you can [enable FedRAMP for some of those agents](#agents). 
\ No newline at end of file
+Notes about FedRAMP compliancy for trace data: 
+* Trace data is reported by some of our agents, like our APM agents, browser agent, and mobile agent. To enable FedRAMP compliancy for that data, you would [enable FedRAMP for the applicable agent](#agents).
+* Currently [Infinite Tracing](/docs/distributed-tracing/infinite-tracing/introduction-infinite-tracing/) is not FedRAMP compliant. 
+
+To ensure FedRAMP compliance for data sent via the [Trace API](/docs/distributed-tracing/trace-api/introduction-trace-api/) (including [telemetry integrations](/docs/integrations/open-source-telemetry-integrations/) that use this API), replace the `https://trace-api.newrelic.com/trace/v1` endpoint with `https://gov-trace-api.newrelic.com/trace/v1`. 
```

Comment:
I wonder if this final sentence should be made into a third bullet, since the agents also send trace data to the Trace API. Maybe you need to clarify that this is for non-agent apps. How about something like this?

* To ensure FedRAMP compliance if you have custom tracing apps or telemetry integrations that send data to the Trace API, replace the endpoint `https://trace-api.newrelic.com/trace/v1` with `https://gov-trace-api.newrelic.com/trace/v1`.

Also, that table of contents page for the telemetry integrations isn't very user-friendly. Although our main page telemetry page isn't a lot better, it's at least a bit more conventional: /docs/integrations/open-source-telemetry-integrations/get-started/introduction-new-relics-open-source-telemetry-integrations. I could go either way on that.

Reply:
actually the agents don't use the API path, in my understanding. But I"m not exactly clear on that. My understanding is that they have their own path, which is why traces sent via the agents were previously fedramp compliant before the Trace API (and things that use it) were fedramp compliant.

Reply:
but maybe that's what you were saying, actually.